### PR TITLE
Override default moment relative time - Arabic, Brasil and Mundo

### LINKED ISSUE #2151: Override moment relative timestamp translations - Arabic, Brasil, Mundo
**Is your feature request related to a problem? Please describe.**
Apply the correct wording for relative timestamp to match what we have currently on PAL.

**Describe the solution you'd like**
Override moment's default relative time in `psammead-locales` to have the correct translations. 
**Services**
- Arabic
- Brasil
- Mundo

**Example**
```js
moment.updateLocale('yo', {
  relativeTime: {
    past: '%s sẹ́yìn',
    s: 'ìṣẹ́jú aayá die',
    m: 'ìṣẹ́jú kan',
    mm: 'ìṣẹ́jú %d',
    h: 'wákàtí kan',
    hh: 'wákàtí %d',
  },
});
```
More details on the spreadsheet. 
**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #2151

**Overall change:** 
Override default moment relative time so that we can have the correct localized relative time.

**Code changes:**
- Add `es` locale for Mundo
- Override moment's relative time for `ar` (Arabic) and `pt-br` (Brasil)

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
This all looks good to me.

File: packages/utilities/psammead-locales/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.10.0 | [PR#2176](https://github.com/bbc/psammead/pull/2176)  Add `es` (Spanish) locale and update `ar` (Arabic) and `pt-br` (Brasil) locales |
```
Comment (sareh) [inline packages/utilities/psammead-locales/CHANGELOG.md]: Can you please also update the [Readme.md](https://github.com/bbc/psammead/tree/a81b812f2b7eb49bd5f33ac8642e32433ea3e186/packages/utilities/psammead-locales#moment-locales) to include the `es` locale?

Comment (DenisHdz) [reply packages/utilities/psammead-locales/CHANGELOG.md]: Sure, I missed it.

File: packages/utilities/psammead-locales/moment/ar.js [hunk 2]
```diff
@@ -27,4 +27,11 @@ moment.updateLocale('ar', {
     // Suppress conversion to Eastern numerals from base `ar` locale but retain use of Arabic comma
     return string.replace(/,/g, '،');
   },
+  relativeTime: {
+    past: '%s',
+    m: 'قبل دقيقة واحدة',
```
Comment (sareh) [inline packages/utilities/psammead-locales/moment/ar.js:32]: I believe moment expects to return a function here. 
https://github.com/moment/moment/blob/develop/src/locale/ar.js#L103
This is the what the value is for `m` in moment config:

```
m: function (number, withoutSuffix, string, isFuture) {
        var f = pluralForm(number),
            str = plurals['m'][pluralForm(number)];
        if (f === 2) {
            str = str[withoutSuffix ? 0 : 1];
        }
        return str.replace(/%d/i, number);
    };
```

Comment (sareh) [reply packages/utilities/psammead-locales/moment/ar.js:32]: Ah, no worries, it seems like that is a special case. Other locales just return strings. https://github.com/moment/moment/blob/develop/src/locale/fa.js#L68

**Review by sareh — APPROVED** [review]
:+1: Looks good!

When reviewing the Arabic locale, it was difficult to use the snapshots from Storybook since the Arabic text is left-aligned and that means a mix of digits 0-9 and Arabic words mess up the word order. We should update the stories for Moment locales to have the `dir={dir} ` prop on the localised text. 
Screenshot Arabic locale story

**Review by dr3 — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**